src/dmd/backend/cod1.d

```diff
@@ -914,6 +914,7 @@ void getlvalue(ref CodeBuilder cdb,code *pcs,elem *e,regm_t keepmsk)
                     break;
 
                 default:
+                    printf("function: %s\n", funcsym_p.Sident.ptr);
```

Comment:
Is this for debugging?  Shouldn't it be commented out?

Reply:
Note line 919